assets/src/edit-story/output/utils/styles.js

```diff
@@ -18,15 +18,40 @@
  * Internal dependencies
  */
 import { FULLBLEED_RATIO, PAGE_RATIO } from '../../constants';
+import theme from '../../theme';
+
+const HEX_CHARS = '1234567890ABCDEF';
+
+function isHexColorString(s) {
+  return (
+    s[0] == '#' &&
+    s
+      .substr(1)
+      .split('')
+      .every((char) => HEX_CHARS.includes(char))
+  );
+}
```

Comment:
I think a regular expression is cleaner here:

```js
function isHexColorString(s) {
  // only allow `#XXX` or `#XXXXXX` with `X` being any hexadecimal character.
  return Boolean(s.match(/^#(?:[a-f0-9]{3}){1,2}$/i));
}

Though I'm not fully sure why we need to do this. Can the theme context be XSS'ed?

Reply:
Also, what if we change it to `rgba(0, 255, 0, 0.5)` ? That's a perfectly legal CSS string too, that we suddenly can't use. Probably not an issue, but still a weird restriction.

Reply:
Cool, regex SGTM. The concern is if unsanitized user input somehow sneaks into variables. Probably low risk but these things can creep in over time.